### PR TITLE
Monica/polishes

### PR DESCRIPTION
**Pull request by myo3**

### CODE REVIEW

File: TCAT.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -837,7 +841,7 @@
 			baseConfigurationReference = 81725F2EBEBD3D184BFD0635 /* Pods-TCAT.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = FV2G57GLFW;
```
Comment (mattbarker016) [inline TCAT.xcodeproj/project.pbxproj]: This has messed me up before, try to leave it the same so we don't get errors when we use master.

File: TCAT.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -661,10 +664,11 @@
 				503EC37F1E969B2100B3D4D4 /* SearchResultsTableViewController.swift in Sources */,
 				DDC83CA31E52B09D00D4EDDF /* LocationTableViewCell.swift in Sources */,
 				DDB49C991E8857E000A99C35 /* SmallDetailTableViewCell.swift in Sources */,
-				DDBFEAA01E787008002BBD96 /* DatePickerView.swift in Sources */,
+				DDBFEAA01E787008002BBD96 /* DatepickerView.swift in Sources */,
```
Comment (mattbarker016) [inline TCAT.xcodeproj/project.pbxproj:667]: Why lowercase now 😥

File: TCAT/Cells/RouteTableViewCell.swift [hunk 3]
```diff
@@ -10,264 +10,206 @@ import UIKit
...
-        departTimeLabel.frame = CGRect(x: 0, y: travelTimeLabel.frame.minY, width: 135, height: 20)
-        departTimeLabel.font = UIFont(name: "SFUIText-Medium", size: 14.0)
-        departTimeLabel.textColor = .tcatBlueColor
-        contentView.addSubview(departTimeLabel)
-        
-        //Set up top seperator line
-        topLine = UIView(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: cellBorderWidthY))
-        topLine.backgroundColor = .lineColor
-        contentView.addSubview(topLine)
+        styleTravelTime()
```
Comment (mattbarker016) [inline TCAT/Cells/RouteTableViewCell.swift:49]: Nice!

File: TCAT.xcodeproj/project.pbxproj (hunk 1 above)
Comment (myo3) [reply TCAT.xcodeproj/project.pbxproj]: good point. ill fix that

File: TCAT.xcodeproj/project.pbxproj (hunk 2 above)
Comment (myo3) [reply TCAT.xcodeproj/project.pbxproj:667]: i like it like that :P

File: TCAT/Controllers/OptionsViewController.swift [hunk 4]
```diff
@@ -223,90 +181,165 @@ class OptionsViewController: UIViewController, UITableViewDelegate, UITableViewD
...
+    }
+    
+    private func showSearchBar(){
+        navigationItem.titleView = searchBarView.searchController?.searchBar
+        searchBarView.searchController?.isActive = true
+    }
+    
     func searchForRoutes(){
-        if searchTime == nil{
+        if searchTime == nil && routeSelection.datepickerButton.titleLabel?.text?.lowercased() == "leave now"{
```
Comment (mattbarker016) [inline TCAT/Controllers/OptionsViewController.swift:279]: Maybe make a safer way to check this?

**Review by mattbarker016 — COMMENTED** [review]
Would appreciate some more eyes on this, it's a lot of code 😅. Seems good though, of course

**Pull request merged**